tests/components/energy/test_sensor.py

```diff
@@ -933,13 +938,22 @@ async def test_cost_sensor_handle_gas_kwh(
     assert state.state == "50.0"
 
 
-@pytest.mark.parametrize("unit", (VOLUME_CUBIC_FEET, VOLUME_CUBIC_METERS))
+@pytest.mark.parametrize(
+    "unit_system,usage_unit,growth",
+    (
+        # 1 cubic foot = 7.47 gl, 100 ft3 growth @ 0.5/ft3:
+        (US_CUSTOMARY_SYSTEM, VOLUME_CUBIC_FEET, "374.025974025974"),
```

Comment:
Is the precision fixed in this string, ie it will not vary?

Reply:
Is it this that decides the precision?

https://github.com/home-assistant/core/blob/0bfb0c25f6ff9a108aa9cf71d3001042f49a0380/homeassistant/helpers/entity.py#L53-L55

Can that vary with the system?

Reply:
I think it make sense in this case to change
`assert state.state == growth`

to

`assert state.state == pytest.approx(growth)`

Would you agree?

Reply:
Yeah, but we'd need to convert them to float first, I think.

Reply:
pushed [99d0cac](https://github.com/home-assistant/core/pull/81379/commits/99d0cac4bbb5f27b967b231721a3db59eccc73da)